src/context/moviesContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {createContext, useContext, useState} from 'react';
```

Comment:
Why are you using context instead of redux?

Reply:
It's not a problem, just wanted to know the reason behind your choice

Reply:
I think Redux would be a bit much in this case. Since the amount of data is small and it's not going to get any bigger, I think it would just make things more complex for no reason.

Reply:
Cool! Sounds reasonable for me